template/CHART_NAME/templates/secret.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: %%COMPONENT_NAME%%
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  name: {{ template "common.names.fullname" . }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+type: Opaque
+data:
+  foo: bar
```

Comment:
Should we use sth different here?

Reply:
What do you suggest? In the end, the data are pairs of key/value that are going to depend on the specific use case

Reply:
Idk.. sth like:
```
  password-key: b64-password-value
```